Automattic/wp-calypso
# ReauthRequired: remove legacy createReactClass

Converts the `ReauthRequired` component from `React.createReactClass` to a modern class component 🙂 

The `observe` mixin is replaced by manually adding/removing a `twoStepAuthorization` listener in lifecycle methods. This event-emitter store should be replaced with something Redux-like in near future.

**How to test:**
Verify that the reauth dialog you see when entering the `/me` section works correctly. You can trigger it by removing the `twostep_auth` cookie.
